diagnostic.md

```diff
@@ -92,7 +122,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+Yellow text, sans-serif cursive font
```

Comment:
There is inline styling the takes precedent over the stylesheet specifying that the font-family will be cursive